content/content-strategy/define-user-content-needs/create-actionable-artefacts/journey-maps/index.yml

```diff
@@ -0,0 +1,29 @@
+pagetitle: Journey maps
+description: Create a journey map after your user research sessions. You will get a broad view of the experience and how your content fits into that.
+weight: 10
+layout: layout/topic
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md #methods
+  - toc.md
+  - supporting-takeaway.md
+  - heading.md
+  - text1.md
+  - heading1.md
+  - text2.md
+  - heading2.md
+  - section.md
+  - diagram.md #diagram
+  - heading3.md
+  - section2.md
+  - supporting-takeaway2.md #supporting-takeaway
+  - heading4.md
```

Comment:
One of the symptoms of numbering markdown files: can make them hard to know what's in them and to verify correct list order. Was the reorder of 4 and 5 intentional?

Reply:
Steve. All numbering fixed.